THE (not so complete) BLOGGING UPDATE

Icey67 opened the pull request:
# AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
The following changes have been made:
- Now using Jekyll as a blogging platform! (I don't know if it'll actually build the blogs as I intend them to)
  - Not that it'll be used much anyway because I currently set the ONE blog to unpublished, please don't reverse-engineer this and publish something incomplete ;-;
  - Also: with a unique style and execution :P
- Different Style?!
  - Now I know I called the last update "monostyle" BUT I think the gradient borders look better (maybe)
- New Footer Message system
  - Allowing HTML tags in the footer for ease of implementation and access
- :shipit:
- New easter egg?
  - you can figure it out

Icey67 commented:
> OHMYGAD IT WORKS

![17275865138036234809550435629944](https://github.com/user-attachments/assets/ec97fe4b-b2d0-4679-be17-1a386ed69783)

Cannot observe how it works since I also just woke up

Icey67 commented:
And with that you are free to merge it :shipit:

eveee00 commented:
wooooo

eveee00 commented:
goddamnit, the only issue this now has is that the text is snapping to the left.

Icey67 commented:
![Screenshot_20240929-072259_Brave](https://github.com/user-attachments/assets/32be6898-00be-46c0-8796-deae8ae8eab0)
Looks good tho 👍

eveee00 commented:
thanks!

Icey67 commented:
though I'd still recommend using this :P
![Screenshot_20240929-072523_Brave](https://github.com/user-attachments/assets/2dec3a24-2ea4-4cf0-9fe1-c503c1c8ac81)

eveee00 commented on style.css:
```diff
@@ -119,33 +120,39 @@ body {
 }
-
-
+/* is this even still necessary? -i67; remov ing the code didn't fuck anything up, so no.*/
```
epic space

Icey67 replied:
![1727587978639284030819167234473](https://github.com/user-attachments/assets/c8a1b4b0-c18b-4c71-8fb4-010a1684a0ea)

Icey67 committed:
BLOGGING UPDATE SOON

i just don't know how to work with jekyll
and with github pages for that matter

Icey67 committed:
WHAT IN THE EVER LOVING FU-

lore and ✨ *style* ✨ added.

Icey67 committed:
WHAT IN THE EVER LOVING FU-

lore and ✨ *style* ✨ added.

Icey67 committed:
short

aeiou

Icey67 committed:
hmmm

i don't know what to call this commit, i just wanna start coding on my pc again

Icey67 committed:
holy moly when will the commits stop

EL MUSICA Added to blogs
and whatever, again same spiel with computer and laptop

Icey67 committed:
am i working hard or am i hardly working?

i'm trying my best ;-;

Icey67 committed:
FSKJLFJLKSDJFKSLDJFKLSJFLKSJ

Implemented @eveee00 's pull request and extended the blog

Icey67 committed:
aeiou?

ehhhhhhhhhhhhhhhhhhh idk

Icey67 committed:
the i am done update

RAHFGUIERWHJFUIFJEWUIJFEWIOJFW

Icey67 commented:
:shipit:
